File: src/main/java/fi/aalto/cs/apluscourses/utils/parser/NewsParser.java
package fi.aalto.cs.apluscourses.utils.parser;

import org.jetbrains.annotations.NotNull;
import org.jsoup.nodes.Element;

public class NewsParser {
  @NotNull
  public String parseTitle(@NotNull Element titleElement) {
    return titleElement.text();
  }

  @NotNull
  public String[] parseBody(@NotNull Element bodyElement) {
    return new String[] {bodyElement.text()};
  }
}

File: src/main/kotlin/fi/aalto/cs/apluscourses/utils/parser/NewsParser.kt
package fi.aalto.cs.apluscourses.utils.parser

import org.jsoup.nodes.Element

open class NewsParser {
    open fun parseTitle(titleElement: Element): String {
        return titleElement.text()
    }

    open fun parseBody(bodyElement: Element): String {
        return bodyElement.html()
    }
}
